raiden/transfer/state.py

```diff
@@ -1003,6 +1003,13 @@ def __eq__(self, other):
             self.deposit_block_number == other.deposit_block_number
         )
 
+    def __lt__(self, other):
```

Comment:
Question: The order we impose here does not really matter, right? It just needs to have the `__lt__` operator implemented to solve ordering for `TransactionChannelNewBalance` as per #1796?

Reply:
Yes, the order doesn't matter. I am not sure why the heap can't order without the '<' operator implemented - maybe `__eq__()` overrides it?

For any generic class the ordering works (and uses instance address as the key)
```
>>> class X():
...   pass
... 
>>> a = X()
>>> b = X()
>>> a < b
True
>>> b < a
False
```

Reply:
Also, please use [`total_odering`](https://docs.python.org/3/library/functools.html?highlight=total_order#functools.total_ordering)

Reply:
nice decorator, thanks for the tip. I wonder why do they require `__eq__` to be implemented if `__lt__` is available.

```
def __eq__(self, o):
  return (not (self < o) ) and (not (o < self))
```